### ISSUE TITLE
DS4 Bluetooth latency issues 

### ISSUE THREAD

**Issue by silverxd** [Ryochan7/DS4Windows#1217]
Hello, thank you for taking your time to look at my issue. I have a very weird problem with DS4Windows. I use a TP-LINK UB400 Bluetooth adapter plugged to a USB2 slot to connect my controller (original DS4) to my computer (win10). I tried using the controller wired and everything worked perfectly. However, upon connecting it to my computer wireless, I started noticing huge input delays, sometimes close to a second. The program was constantly reporting delays over 20ms. Changing the polling rate to a higher value seemed to temporarily fix the issue, but now it's back and even worse than before. It seemed to me like an interference issue, but there is almost no interference anywhere near the computer and dongle. What's weird is that an Xbox one controller works just fine using that same dongle (I feel no delay at all, just like using DS4 wired), so it's probably not the dongle's fault. I'll provide a log file later today. 

Basically my main question is: what is probably the cause of the Bluetooth delay (only happens on DS4) and how can I fix it?

EDIT: log added
[log.txt](https://github.com/Ryochan7/DS4Windows/files/4464500/log.txt)

**Comment by silverxd** [comment]
Included screenshots of my profiles and settings.
![DS4Windows_7WwoaOtbmO](https://user-images.githubusercontent.com/57298235/79042317-620b3500-7bff-11ea-831c-cb0e6a2d7031.png)
![DS4Windows_b1vrmV1GJp](https://user-images.githubusercontent.com/57298235/79042318-62a3cb80-7bff-11ea-8101-3ed95a88be9e.png)
![DS4Windows_uJB7n2pe8u](https://user-images.githubusercontent.com/57298235/79042319-633c6200-7bff-11ea-9a95-3fde60861f2a.png)

**Comment by mika-n** [comment]
These BT lag issues are sometimes a mystery.
https://github.com/Ryochan7/DS4Windows/wiki/Troubleshooting#gamepad-connection-over-bluetooth-bt

**Comment by silverxd** [comment]
Thanks for the reply. I have already done everything suggested in that article way back in January. What's weird is that after I posted my issue, the delay has suddenly disappeared. After about an hour, it came back, but then disappeared again. Indeed, it is a mystery. I suppose I'll live on with it.
